build.sbt

```diff
@@ -64,7 +64,7 @@ libraryDependencies ++= Seq(
 )
 // @formatter:on
 
-resolvers ++= Seq("segence" at "https://dl.bintray.com/segence/maven-oss-releases/")
+//resolvers ++= Seq("segence" at "https://dl.bintray.com/segence/maven-oss-releases/")
```

Comment:
Is this meant to be commented out? Can it be removed?

Reply:
[`c86d840` (#54)](https://github.com/sky-uk/kafka-topic-loader/pull/54/commits/c86d8406a8f39350e289cdd5f7d3cd5a0fbd779e)